client/my-sites/customer-home/cards/tasks/go-mobile/index.js

```diff
@@ -42,9 +43,11 @@ const GoMobile = ( { isIos } ) => {
 
 	return (
 		<Task
-			title={ translate( 'Update and manage on the go' ) }
-			description={ translate(
-				'Inspiration strikes any time, anywhere. Post, read, check stats, and more with the Wordpress app at your fingertips.'
+			title={ preventWidows( translate( 'Update and manage on the go' ) ) }
+			description={ preventWidows(
+				translate(
+					'Inspiration strikes any time, anywhere. Post, read, check stats, and more with the WordPress app at your fingertips.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.